web/server/www/scripts/codecheckerviewer/CommentView.js

```diff
@@ -121,31 +120,25 @@ function (declare, dom, style, topic, Memory, Observable, ConfirmDialog,
       });
 
       this._commentContent = new SimpleTextarea({
-        value : that.message
+        value : this.commentData.message
       });
+      this._editDialog.addChild(this._commentContent);
 
       this._saveButton = new Button({
         label : 'Save',
         onClick : function () {
           var newContent = that._commentContent.get('value');
 
           try {
-            CC_SERVICE.updateComment(that.cId, newContent);
+            CC_SERVICE.updateComment(that.commentData.id, newContent);
           } catch (ex) { util.handleThriftException(ex); }
 
-          that._message.set('content',
-            newContent.replace(/(?:\r\n|\r|\n)/g, '<br>'));
+          that._message.innerHTML =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unsafe assignment to innerHTML (no-unsafe-innerhtml/no-unsafe-innerhtml)](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=3374942)